src/test/java/seedu/address/storage/JsonMasterDeckStorageTest.java

```diff
@@ -80,7 +80,7 @@ public void readAndSaveMasterDeck_allInOrder_success() throws Exception {
         assertEquals(original, new MasterDeck(readBack));
 
         // Save and read without specifying file path
-        original.addCard(FRACTION);
+        original.addCard(GRAVITY);
```

Comment:
Changed card to Gravity because Fraction belongs to a deck that was not initialized. The problem is addCard does not add a deck if it doesnt exist,